diff --git a/dist/client/client/theme/SearchBar/SearchBar.jsx b/dist/client/client/theme/SearchBar/SearchBar.jsx
index 14f54733d9e4851dd7f82ccd8fcb1b2896632b70..4a2f76701c8efedd28c5c87b2897212706eafc36 100644
--- a/dist/client/client/theme/SearchBar/SearchBar.jsx
+++ b/dist/client/client/theme/SearchBar/SearchBar.jsx
@@ -395,7 +395,11 @@ export default function SearchBar({ handleSearchBarToggle, }) {
             id: "theme.SearchBar.label",
             message: "Search",
             description: "The ARIA label and placeholder for search button",
-        })} aria-label="Search" className={`navbar__search-input ${styles.searchInput}`} onMouseEnter={onInputMouseEnter} onFocus={onInputFocus} onBlur={onInputBlur} onChange={onInputChange} ref={searchBarRef} value={inputValue}/>
+        })} aria-label={translate({
+            id: "theme.SearchBar.label",
+            message: "Search",
+            description: "The ARIA label and placeholder for search button",
+        })} className={`navbar__search-input ${styles.searchInput}`} onMouseEnter={onInputMouseEnter} onFocus={onInputFocus} onBlur={onInputBlur} onChange={onInputChange} ref={searchBarRef} value={inputValue}/>
       {askAi && AskAIWidgetComponent && (<AskAIWidgetComponent ref={askAIWidgetRef} {...askAi}>
           <span hidden></span>
         </AskAIWidgetComponent>)}
